backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -60,12 +62,15 @@
     private static final String USER_ALREADY_FOLLOWED = "User already followed";
```

Comment:
Yeah same thing here but this is from another story. It probably slipped through my eyes. :P